Workers/DatabaseSyncWorker/DatabaseChanges.cs

```diff
@@ -60,7 +69,10 @@ public DatabaseChanges(PostgresOptions postgresOptions, string tableName)
                     }
                     catch (Exception e)
                     {
-                        Log.Error(e, "LISTEN {TableName}", tableName);
+                        if (!tcs.Token.IsCancellationRequested)
```

Comment:
Is this to filter out cancelled exceptions? If so you could pattern match on the type of `e` here

Reply:
This is to filter out exceptions that are thrown as a result of cancelling, the catch block above should catch the explicit `TaskCanceledException` type